main/station-config.js

```diff
@@ -40,53 +42,93 @@ const StationID = /** @type {string} */ (configStore.get(ConfigKeys.StationID, r
 /**
  * @returns {boolean}
  */
-function getOnboardingCompleted () {
+function getOnboardingCompleted() {
   return !!OnboardingCompleted
```

Comment:
Should these variables be `camelCase` instead of `PascalCase`? I'm picking this up because with pascal case I would expect them to be classes or constructors.